CMakeLists.txt

```
@@ -133,6 +133,23 @@ ecbuild_add_option( FEATURE FIELD_API
                     DEFAULT ON
                     REQUIRED_PACKAGES "field_api" )
 
+ecbuild_add_option( FEATURE FIELD_API_ENABLE_IO
+                    DESCRIPTION "Use field_api IO to store field_api objects to disk."
+                    CONDITION HAVE_FIELD_API
+                    DEFAULT OFF )
+
+if( HAVE_FIELD_API_ENABLE_IO )
+   list(APPEND CLOUDSC_DEFINITIONS FIELD_API_IO)
+   if( HAVE_FIELD_API_ENABLE_IO_MPL )
+     list(APPEND CLOUDSC_DEFINITIONS FIELD_API_IO_MPL)
+   elseif( HAVE_FIELD_API_ENABLE_IO_MPI )
+     find_package(MPI REQUIRED )
```

I think it makes more sense to check here that the MPI feature is enabled. The bundle implicitly enforces this but a CMake-build without bundle would allow a halfway-house otherwise

```suggestion
     if( NOT HAVE_MPI )
         ecbuild_error("Cannot enable FIELD_API_ENABLE_IO_MPI when feature MPI is disabled.")
     endif()
```

---

Actually, at the moment it is activated in FieldAPI itself:
elseif(HAVE_IO_MPI)
ecbuild_enable_mpi( COMPONENTS FORTRAN REQUIRED)

so this section of cloudsc can be deleted indeed.
However, if we link to parallel version of HDF5, it is probably necessary anyway to activate MPI, even without --with-mpi in the bundle.